lib/Github/Client.php

```diff
@@ -127,7 +127,7 @@ public function __construct(Builder $httpClientBuilder = null, $apiVersion = nul
         )));
 
         $this->apiVersion = $apiVersion ?: 'v3';
-        $builder->addHeaders(['Accept' => sprintf('application/vnd.github.%s+json', $this->apiVersion)]);
+        $builder->addHeaderValue('Accept', sprintf('application/vnd.github.%s+json', $this->apiVersion));
```

Comment:
Otherwise we wouldn't be able to add the custom `Accept` header value without having to repeat `Accept` header values passed by this builder.